lxd/storage/drivers/utils.go

```diff
@@ -765,7 +765,9 @@ func BTRFSSubVolumeIsRo(path string) bool {
 	return strings.HasPrefix(string(output), "ro=true")
 }
 
-// BTRFSSubVolumeMakeRo makes a subvolume read only. Deprecated use btrfs.setSubvolumeReadonlyProperty().
+// BTRFSSubVolumeMakeRo makes a subvolume read only.
+//
+// Deprecated: Use btrfs.setSubvolumeReadonlyProperty().
```

Comment:
Can we remove this function and update the callers to use btrfs.setSubvolumeReadonlyProperty?